src/main/java/com/example/fooddelivery/repository/UserRepository.java

```diff
@@ -1,7 +1,10 @@
 package com.example.fooddelivery.repository;
 
+import org.springframework.data.jpa.repository.JpaRepository;
+
 import com.example.fooddelivery.model.User;
-import org.springframework.data.repository.CrudRepository;
 
-public interface UserRepository extends CrudRepository<User, Long> {
+public interface UserRepository extends JpaRepository<User, Long> {
```

Comment:
Why you need `JpaRepository` instead of `CrudRepository`? Currently, we don't use any feature of this, like pagination for example. Also, `JpaRepo` extends from `CrudRepo`.